### DISCUSSION TITLE
Nested albums - album inside album

### DISCUSSION THREAD

**Discussion by fr-gg** [ente-io/ente#928] [Enhancements]
Multiple albums in one album would be a really useful feature to organize photos. AFAIK this isn't possible currently.

**Comment by vishnukvmd** [comment]
Duplicate: https://github.com/ente-io/ente/discussions/782

**Comment by brog-io** [comment]
This should probably be closed in favor of https://github.com/ente-io/ente/discussions/552

**Reply by vishnukvmd** [reply]
Github unfortunately does not allow merging discussions, so we'll lose the upvotes :(

But yes, given #552 has more traction, I'll close this in favor of that.
